### ISSUE TITLE
The database suddenly shutting down，I don’t know why. Here is the log. Can anyone help me take a look?

### ISSUE THREAD

**Issue by alvin22** [greenplum-db/gpdb#16598]
![企业微信截图_16975077891930](https://github.com/greenplum-db/gpdb/assets/9802062/da91eba3-17f7-49f4-b182-9220d37eecfb)

**Comment by alvin22** [comment]
The version information is  PostgreSQL 9.4.24 (Greenplum Database 6.16.3 build commit:f245f5686a407beb99f46a9ac7e3b9152223724b Open Source) on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 6.4.0, 64-bit compiled on Jul  2 2021 01:31:17

**Comment by sokolval** [comment]
It seems that logs you provided are not enough to understand the reason of DB Shutdown
The statement received smart shutdown request on top seems to indicate, that someone triggered gpstop on master host

**Comment by Aegeaner** [comment]
The error code `57P03` means `ERRCODE_CANNOT_CONNECT_NOW`, implies failure with database connections.

**Comment by alvin22** [comment]
> The error code `57P03` means `ERRCODE_CANNOT_CONNECT_NOW`, implies failure with database connections.

Yes, I know, but the database crashes almost once a month, but I don’t know the reason.

**Comment by alvin22** [comment]
> It seems that logs you provided are not enough to understand the reason of DB Shutdown The statement received smart shutdown request on top seems to indicate, that someone triggered gpstop on master host

Yes, I know, but the database crashes almost once a month, but I don’t know the reason.

**Comment by kainwen** [comment]
6.16.3 is much too old. You should upgrade to recent version of GPDB.
